task/terraform-executor/0.1/terraform-executor.yaml

```diff
@@ -16,6 +16,8 @@ spec:
     The usual actions are `plan` and `apply`. Authentication to Google Cloud is done through the Kubernetes service account with the
     workload identity configured. The parameter **additional_terraform_args** allows defining Terraform CLI arguments in pipelines.
     When this parameter value is set in a pipeline, you must add the default value `-no-color` to the list.
+    The parameter **additional_tfcmt_args** allows defining additional variable tfcmt CLI arguments in pipelines.
+    This parameter must at least provide -sha and/or -pr arguments together with its values. See example pipeline for parameters values example.
```

Comment:
I'm not sure I understand this line correctly, but I think that, depending on the number of parameters in question, either ` This parameter must at least provide -sha and/or -pr arguments together with its values. See the example pipeline for the parameter's values example.`, or ` This parameter must at least provide -sha and/or -pr arguments together with its values. See the example pipeline for the parameters' values example` should sound better.